src/Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -919,6 +909,19 @@ public bool IsLinkLocalAddress(IPAddress address)
         return NetworkConstants.IPv4RFC3927LinkLocal.Contains(address) || address.IsIPv6LinkLocal;
     }
 
+    private static bool SubNetContainsAddress(IPNetwork network, IPAddress address)
+    {
+        ArgumentNullException.ThrowIfNull(address);
+        ArgumentNullException.ThrowIfNull(network);
+
+        if (address.IsIPv4MappedToIPv6)
+        {
+            address = address.MapToIPv4();
+        }
+
+        return network.Contains(address);
+    }
```

Comment:
Since this is universal, it should be moved to `NetworkUtils`

Reply:
Lets break ABI for next release

Reply:
Since we only add something new I don't see why it should be a problem.